src/pages/p/strava-activities/index.tsx

```diff
@@ -90,9 +95,44 @@ function activityDistance(
 export default function StravaActivities({
   data,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
-  const { garminActivities } = useGarminActivities();
+  const [activities, setActivities] = useState<Activity[]>(data.activities);
+
+  const fetchMore: FetchMore = async ({ pageSize, pageNumber }) => {
+    try {
+      const res = await fetch(
+        `/api/milameter/getActivities?pageSize=${pageSize}&pageNumber=${
+          pageNumber + 1
```

Comment:
Should probably rename from `pageNumber` to `currentPageNumber` to reduce ambiguity